Signal/src/AppDelegate.m

```diff
@@ -239,7 +239,11 @@ - (void)applicationDidBecomeActive:(UIApplication *)application {
                                                [TSSocketManager becomeActiveFromForeground];
                                                [[Environment getCurrent].contactsManager verifyABPermission];
                                            }];
-
+    
+    if ([TSAccountManager isRegistered]) {
+        [[PushManager sharedManager] applicationDidBecomeActive];
```

Comment:
I'm not convinced that `PushManager` is the ideal place to hang this method, but it already owns the messageFetcherJob, so it's expedient to do so... for now.